freon@UshakovVasilii_Github.yahoo.com/prefs.js

```diff
@@ -26,82 +26,161 @@ var FreonPrefsWidget = new GObject.registerClass(class Freon_FreonPrefsWidget ex
+        this.attach(panelBoxIndex, j + 1, i++, 1, 1);
         this._settings.bind('panel-box-index', panelBoxIndex, 'value', Gio.SettingsBindFlags.DEFAULT);
 
-        this._addSwitch({key : 'show-degrees-on-panel', y : i++, x : 3,
-            label : _('Show \u00b0C/\u00b0F on Panel')});
+        this._addSwitch({key : 'show-icon-on-panel', y : i++, x : j,
+            label : _('Show Icon')});
 
-        this._addSwitch({key : 'show-icon-on-panel', y : i++, x : 3,
-            label : _('Show Icon on Panel')});
+        this._addComboBox({
+            label: 'Temperature Unit',
+            items : {centigrade : "\u00b0C", fahrenheit : "\u00b0F"},
+            key: 'unit', y : i++, x : j
+        });
 
-        this._addSwitch({key : 'show-fan-rpm', y : i, x : 0,
-            label : _('Show Fan Speed')});
+        this._addSwitch({key : 'show-degrees-on-panel', y : i++, x : j,
+            label : _('Show Temperature Unit')});
 
-        this._addSwitch({key : 'show-voltage', y : i++, x : 3,
-            label : _('Show Power Supply Voltage')});
+        this._addSwitch({key : 'show-rpm-on-panel', y : i++, x : j,
+            label : _('Show Fan Speed Unit')});
 
-        this._addSwitch({key : 'group-temperature', y : i, x : 0,
-            label : _('Group Temperature Items'),
-            help : _("Works if you have more than three temperature sensors")});
+        this._addSwitch({key : 'show-volts-on-panel', y : i++, x : j,
+            label : _('Show Voltage Unit')});
 
-        this._addSwitch({key : 'group-voltage', y : i++, x : 3,
-            label : _('Group Voltage Items'),
-            help : _("Works if you have more than three voltage sensors")});
+        this._addSwitch({key : 'show-decimal-value', y : i++, x : j,
+            label : _('Show Decimal Value'),
+            help : _("Show one digit after decimal")});
 
-        this._addComboBox({
-            items : {none : _('None'), hddtemp : 'Hddtemp', udisks2 : 'UDisks2', smartctl : 'smartctl', nvmecli : 'nvme-cli'},
-            key: 'drive-utility', y : i, x : 0,
-            label: _('HDD/SSD Temperature Utility')
+        i = 0;
+        j = 3;
+
+        this._addLabel({
+            label: _('Generic sensors'),
+            y : i++, x : j
         });
 
-        this._addComboBox({
-            items : {
-                'none' : _('None'),
-                'nvidia-settings' : _('NVIDIA'),
-                'aticonfig' : _('Catalyst'),
-                'bumblebee-nvidia-smi': _('Bumblebee + NVIDIA') },
-            key: 'gpu-utility', y : i++, x : 3,
-            label: _('Video Card Temperature Utility')
+        this._addSwitch({key : 'use-generic-lmsensors', y : i++, x : j,
+            label : 'lm-sensors',
+            help : _('Read sensors using sensors from lm-sensors')});
+
+        this._addSwitch({key : 'use-generic-liquidctl', y : i++, x : j,
+            label : 'liquidctl',
+            help : _('Read sensors using liquidctl v1.7.0 or later')});
```

Comment:
```suggestion
            help : _('Read sensors from liquidctl (requires v1.7.0 or later)')});
```

This is a precondition we don't check at runtime, so it must be explicitly marked as a requirement.

Reply:
Done.